examples/key_value_store/canisters/rust/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "rust"
+version = "0.0.0"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+ic-cdk = "0.5.2"
+ic-cdk-macros = "0.5.2"
+candid = "0.7.14"
```

Comment:
Do you need candid? And no newline

Reply:
Yes, the perf result uses candid:CandidType

New line added